autoload/go/oracle.vim

```diff
@@ -96,18 +96,24 @@ func! s:RunOracle(mode, selected, needs_package) range abort
     if empty(bin_path) 
         return 
     endif
+    
+    if exists('g:go_oracle_tags')
+        let tags = get(g:, 'go_oracle_tags')
+    else
+        let tags = ""
+    endif
 
     if a:selected != -1
         let pos1 = s:getpos(line("'<"), col("'<"))
         let pos2 = s:getpos(line("'>"), col("'>"))
-        let cmd = printf('%s -format plain -pos=%s:#%d,#%d %s',
+        let cmd = printf('%s -format plain -pos=%s:#%d,#%d -tags=%s %s',
```

Comment:
Did you test how oracle behaves if you pass an empty string `""` to the `-tags` flag?